src/Directory.Build.props

```diff
@@ -9,6 +9,7 @@
     <LangVersion>10</LangVersion>
     <Nullable>enable</Nullable>
     <ImplicitUsings>enable</ImplicitUsings>
+    <AnalysisLevel>latest-all</AnalysisLevel>
```

Comment:
When only targeting netstandard2.0, .NET 6 code analyzers are not enabled. Remove a `.ConfigureAwait(false)` somewhere in the code and you'll see that there is a warning. Remove also this csproj line and there will be no warning...